### PR TITLE
Lockout guides pilot

### PR DESCRIPTION
**Pull request by rachelnabors**
This is a different approach to the User Lockout docs. It:

* Introduces a new Guides section to the docs
* Groups the new content together under a "Job to be done" heading of "Protect accounts from attacks"

The guides section is just underneath the Quickstarts and Tutorials in the main nav (icon requires a push to marketing):
Screenshot 2023-11-23 at 2 15 48 PM

The section is a little scanty for now, but as the content grows, so too shall this landing page:
Screenshot 2023-11-23 at 2 15 37 PM

See them here:
* [Guides landing page](https://docs-preview-512.clerkpreview.com/docs/guides/overview)
* [Brute force attacks](https://docs-preview-512.clerkpreview.com/docs/guides/user-lock-guide)
* [Customize the max attempts before getting locked out and duration of lock out](https://docs-preview-512.clerkpreview.com/docs/guides/customize-user-lockout)
* [Unlock user accounts from the Clerk Dashboard](https://docs-preview-512.clerkpreview.com/docs/guides/unlock-user-accounts)
* [Programmatically locking and unlocking user accounts](https://docs-preview-512.clerkpreview.com/docs/guides/programatically-lock-user-accounts)

### CODE REVIEW

File: docs/custom-flows/error-handling.mdx [hunk 1]
```diff
@@ -97,4 +97,31 @@ const signIn = await client.signIn.create({
...
+            "code": "user_locked",
+            "meta": {
+                "duration_in_seconds": 1800
+            }
+        }
+    ]
+}
+```
+
+Note: Actual remaining duration will vary depending on your configuration.
```
Comment (yourtallness) [inline docs/custom-flows/error-handling.mdx:125]: Let's add a link to the final configuration docs location.

File: docs/guides/customize-user-lockout.mdx [hunk 2]
```diff
@@ -0,0 +1,35 @@
+---
+title: Customize max login attempts and duration of user lockout
+description: 
+---
+
+# Customize the max attempts before getting locked out and duration of lock out
+
+<Callout type="info">
+This feature is only available for user accounts that use [passwords](https://dashboard.clerk.com/last-active?path=user-authentication/email-phone-username) and [backup codes](https://dashboard.clerk.com/last-active?path=user-authentication/multi-factor).
```
Comment (yourtallness) [inline docs/guides/customize-user-lockout.mdx:9]: "Or" backup codes (or both)

Comment (yourtallness) [reply docs/guides/customize-user-lockout.mdx:9]: Since we don't conditionally disable User Lockout if neither password or backup code is available, I'd rephrase this part to:

>This feature is applicable to user accounts that use passwords or backup codes.

**Review by yourtallness — APPROVED** [review]
Other than a few copy suggestions, looks good to me.

Note: we'll need to update the screenshots for the Attack Protection page, because it's undergoing a revamp.

File: docs/custom-flows/error-handling.mdx (hunk 1 above)
Comment (rachelnabors) [reply docs/custom-flows/error-handling.mdx:125]: Where is the canonical location?

Comment (rachelnabors) [reply docs/custom-flows/error-handling.mdx:125]: Ah, you mean the customization section

**Comment by rachelnabors** [comment]
User testing showed learners are more likely to look for these guides in "Security." Hence, we are for updating this PR to put the new guides under that heading instead of in a new Guide section:

Screenshot 2023-11-27 at 3 57 04 PM

**Comment by rachelnabors** [comment]
Going to close this PR since we're moving them under **Security** in another PR
